python/sglang/srt/managers/io_struct.py

```diff
@@ -469,6 +469,26 @@ class GetWeightsByNameReqOutput:
     parameter: list
 
 
+@dataclass
+class ReleaseGPUOccupationReqInput:
```

Comment:
Call it `ReleaseMemoryOccupationReqInput`. Rename all places (code, api name, etc).